engine/src/test/java/io/camunda/zeebe/process/test/engine/EngineStateMonitorTest.java

```diff
@@ -75,57 +76,60 @@ void testCallbackIsCalledWhenEngineIsAlreadyBusy()
   }
 
   @Test
-  void testCallbackIsCalledWhenBusyStateIsReached()
+  void testOnProcessingCallbackIsCalledWhenBusyStateIsReached()
```

Comment:
:x: If I am not mistaken this test now has exactly the same code as `...IsAlreadyBusy`